src/components/frame/v5/pages/VerifiedPage/VerifiedPage.tsx

```diff
@@ -19,7 +24,11 @@ const VerifiedPage: FC = () => {
       <TwoColumns aside={<Navigation pageName="members" />}>
         <div className="flex justify-between mt-6">
           <div className="w-full">
-            <Table list={verifiedMembers} onReputationSortClick={() => {}} />
+            <Table
```

Comment:
Maybe you should use here the same table which we used in actions? 
`import Table from '~v5/common/Table'; `